Feature/generate fake data

this-kramer opened the pull request:
## Reason for this PR:
 - We would like to present our frontend with somehow realistic looking data (Not Luke Skywalker 1...100 or so)
 - Creating this bootstrap data by hand is boring

## Changes in this PR:
 - Add python script that generates Users (Admin, Lecturer, Student), Examregs (including Modules) and Courses
 - Include some generated files for reviewing :)

## Notes:
 - We might want to delete the generated folder or move the files to some other direction

this-kramer committed:
Add script for generating fake data.

this-kramer committed:
Generate fake entries and fix some errors in script.

this-kramer committed:
Merge develop

this-kramer committed:
Refactor generate fake data script, change passwords to match usernames

this-kramer committed:
Generate some fake data.

this-kramer committed:
Prettyprint json

this-kramer committed:
Reformat script

david-buderus reviewed (changes requested):
The files should be dumped into the right folder or the seeding script should check the generated folder too.
Last one would be better, so you can distinguish between generated and manual created examples.

this-kramer committed:
Fix some api spelling mistakes

this-kramer committed:
Minor fixes and remove some of the generated files

this-kramer committed:
Add generated files to add defaults script

this-kramer committed:
Remove leading zeros from house numbers and generate some data

this-kramer commented:
> The files should be dumped into the right folder or the seeding script should check the generated folder too.
> Last one would be better, so you can distinguish between generated and manual created examples.

Added this to the script and fixed some small api errors. I tried this on develop and it worked
